src/lib/encoding.ts

```diff
@@ -231,6 +231,7 @@ export function parseManifestStream(stream: Readable): EventEmitter {
   let currentKey: string | undefined;
   const keyPath: Array<string | number> = [];
   let indexPath: string | undefined;
+  let wildcardpath: string | undefined;
```

Comment:
Nit: keep variable casing consistent with rest of the codebase, i.e. `wildcardPath`

Reply:
Actually maybe this is an error since you're using `wildcardPath` below but `wildcardpath` here? 🧐